.github/workflows/javadoc.yml

```diff
@@ -0,0 +1,34 @@
+name: Maven Site
+
+on:
+  push:
+    branches: [ main ]
+  pull_request_target:
```

Comment:
javadoc flow is only triggered when pushing on main directly or PR merged. This line and following lines (types, branches) should be deleted.

Reply:
Thanks @shrutikumar15 for updating but I was talking about deleting from line 6 to 8 . Sorry, I should have been more precise. Anyway, I will add this:

```yaml
on:
  push:
    branches: [ main ]
```